core/container/src/main/java/org/wildfly/swarm/internal/MavenArgsParser.java

```diff
@@ -0,0 +1,54 @@
+package org.wildfly.swarm.internal;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.StringTokenizer;
+
+/**
+ * Created by hbraun on 22.08.17.
+ */
+public class MavenArgsParser {
```

Comment:
In addition to `-f`, you can also do `--file`.

Also you can do `-fanotherpom.xml` and it works too. This only applies to the single-character option, for `--file`, there needs to be a space.